.mocharc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should delete this file (there is a `.mocharc.js` )

Reply:
deleted. thanks.